dist/formats/email_alert_signup/publisher/schema.json

```diff
@@ -139,6 +139,13 @@
       "type": "object",
       "additionalProperties": false,
       "properties": {
+        "parent": {
+          "type": "array",
+          "maxItems": 1,
```

Comment:
We should definitely be using the existing definitions (see any other format with links for example).

Also, "parent" is probably right name.  We've already used that for topics - https://github.com/alphagov/govuk-content-schemas/blob/master/formats/topic/publisher/links.json